src/App.tsx

```diff
@@ -3,30 +3,39 @@ import { QueryClient, QueryClientProvider } from "react-query";
 import { ReactQueryDevtools } from "react-query/devtools";
 import "./App.css";
 import logo from "./logo.svg";
+import { theme } from "./theme";
+import { CustomLocalizationProvider } from "./common/components/CustomLocalizationProvider";
+import { CssBaseline } from "@mui/material";
+import { ThemeProvider } from "@mui/private-theming";
```

Comment:
I think it's better to import all from `@mui/material`
```suggestion
import { ThemeProvider } from "@mui/material";
```

Reply:
Fixed in [ed39ed3](https://github.com/jakkapanksb/fe-template/pull/2/commits/ed39ed39a9b044ecb0d61489d706aa42a3d7ef47)